cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -158,6 +170,29 @@ B-tree indexes with the `lucene+native-3.0` index provider and no index configur
 CREATE TEXT INDEX ...
 ----
 
+a|
+label:functionality[]
+label:deprecated[]
+[source, cypher, role="noheader"]
+----
+CREATE [BTREE] INDEX
+...
+OPTIONS "{" indexProvider: 'lucene+native-3.0', indexConfig: btree-spatial-setting[, ...] "}"
+----
+a|
+B-tree indexes are deprecated and need to be migrated before upgrading to 5.0.
+
+B-tree indexes with the `lucene+native-3.0` index provider and spatial index configuration settings are replaced by two indexes:
+[source, cypher, role="noheader"]
+----
+CREATE TEXT INDEX ...
+----
+and
+[source, cypher, role="noheader"]
+----
+CREATE POINT INDEX ...
+----
```

Comment:
should this be replaced by range + text + point, since point and text don't cover all the property values?

Reply:
Possibly, maybe we should ask someone from kernel to confirm all the index deprecations on this page?

Reply:
For that matter, should the `lucene+native-3.0` only case be replaced by text + range...

Reply:
yeah so I made a comment on the first one I saw higher up. it is a bit of a mess depending on if user knows what they are doing or not.
But I guess the simple way of looking at it is:
If they want to do spatial queries -> point index
If they want to do string queries -> text index
If they index other types or not interested in special queries -> range index
And any combos of those possible of course.

Reply:
I agree that the deprecations are probably correct but not written in a very user friendly way. It would be good to reorganize this page to make it more pedagogical. Like first saying the BTREE is deprecated and then list the usecases Emma listed above and for each say what new command we have added. Maybe a bit hard to fix before we know which command have planner support and not though ...